src/oph/ehoks/hoks/hankittavat.clj

```diff
@@ -276,7 +276,8 @@
             existing (db/select-osaamisen-hankkimistavat-by-hoks-id-and-tunniste
                        oh-type
                        hoks-id
-                       (:yksiloiva-tunniste oh))
+                       (:yksiloiva-tunniste oh)
+                       db-conn)
```

Comment:
Täytyy myöntää, etten ihan tajua, mitä hyötyä transaktiosta on tässä (mutta se on toki aina parempi kuin että ei ole transaktiota).

Reply:
Mut se, mikä on vielä hämmentävämpää, on, etten keksi missä tilanteessa existing-logiikalle on käyttöä.
- Jos HOKS luodaan, valmiiksi olemassa olevia OHT:ita ei pitäisi olla, joten existing pitäisi olla tyhjä.
- Jos HOKS korvataan, tässä vaiheessa vanhat tutkinnonosat on jo poistettu, jolloin select-osaamisen-hankkimistavat-by-hoks-id-and-tunniste ei palauta niiden OHT:ita, koska siellä käytetyissä queryissa on aina liitosehtona `hatoosat.deleted_at IS NULL` tai vastaava

Reply:
Oikein hyvä huomio, kiitoksia! Tosiaan oht:n exising-logiikalle ei enää tämän deleted_at-propagoinnin jälkeen ollut tarvetta, joten poistin sen kokonaan.